roles/README.md

```diff
@@ -0,0 +1,54 @@
+<h1 align="center">
+  Stratum Roles
+</h1>
+
+# Introduction
+
+Roles in Stratum are knowns as Job Declarators (JDs), JDs are roles which can be Pool-side and Miner-side, but they can also be run by any third parties. They are connected to a Template Provider, in this way they are able to receive and validate custom block templates. They are the roles needed to implement the so called Job Declaration Protocol. They can further distribute jobs to a Mining Proxy (or Proxies) via the Job Distribution Protocol.
```

Comment:
> Roles in Stratum are knowns as Job Declarators (JDs)
this is inaccurate.

JDs are roles, but not every role is a JD.

please refer to the documentation for the definition:
https://stratumprotocol.org/docs/#roles

Reply:
alright thank you for bringing it to my notice.